README.md

```diff
@@ -13,6 +13,16 @@ See the [`cfn` examples](examples/README.md) to setup prerequisites and get star
 ### MongoDB Atlas AWS CDK
 See the [cdk examples](https://github.com/mongodb/awscdk-resources-mongodbatlas/blob/main/examples/README.md) to setup prerequisites and get started with your first cluster using our AWS CDK sample code.
 
+## Limitations
+1. [Resource import](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import.html) is not supported for third-party resources.
+
+2. CloudFormation support for any third-party resource requires that the third-party resource support all CRUD operations. For this reason, the following resources are not supported: 
```

Comment:
Optional suggestion, this sentence is hard to parse/understand, if this conveys the same meaning you were going for it may be clearer:
```suggestion
2. The following resources are not supported because they require support for all CRUD operations: 
```

Reply:
Good call, this is hard to parse. I want to communicate the general requirement that any third party resource would need to support all CRUD ops to work because these third-party tools change without our knowledge. I'm thinking:

"Any third-party resource you integrate must support all CRUD operations. CloudFormation does not support the following resources because they do not support all CRUD operations:"